packages/contracts-bedrock/src/cannon/MIPS2.sol

```diff
@@ -577,7 +573,11 @@ contract MIPS2 is ISemver {
             } else if (syscall_no == sys.SYS_TIMERDELETE) {
                 // ignored
             } else {
-                revert("MIPS2: unimplemented syscall");
+                if (syscall_no == sys.SYS_FSTAT64 || syscall_no == sys.SYS_STAT64 || syscall_no == sys.SYS_LLSEEK) {
+                    // noop
+                } else {
+                    revert("MIPS2: unimplemented syscall");
+                }
```

Comment:
These could go into the switch-case. But they're no-op'd here so they're similar to the Go VM.